docs/ManualDeploymentGuideScalarDBServerOnEKS.md

```diff
@@ -0,0 +1,199 @@
+# Deploy Scalar DB Server on AWS
+
+The Scalar DB server is a standalone gRPC server that implements the Scalar DB interface.
+We can deploy the Scalar DB server on any Kubernetes service.
+This guide will help you to deploy a Scalar DB server in AWS EKS.
+
+## What we create
+
+![image](images/scalardbserver-eks-diagram.png)
+
+In this guide, we will create the following components.
+
+* A VPC with NAT gateway
+* An EKS cluster with two Kubernetes node groups
+* A managed database service supported by Scalar DB.
+* A Bastion instance with a public IP
+* Amazon CloudWatch
+
+
+## Step 1. Configure your network
+
+Configure a secure network according to your organizational standard or application requirements. This section shows how to configure a secure network for Scalar DB server deployment.
+
+### Requirements
+
+* You must create VPC with public NAT gateways on private networks. NAT gateway is necessary to enable internet access for Kubernetes node group subnets.
+  * Internet access is required to pull Scalar DB server image for Kubernetes deployment.
```

Comment:
Bullet points are not supposed to be used to separate sentences.
If this is the reason for the first sentence, you can just append in the first sentence.

Reply:
Thank you.
I have updated it and added it in the first sentence itself.